src/Symfony/Component/Workflow/WorkflowInterface.php

```diff
@@ -42,6 +44,16 @@ public function getMarking($subject);
      */
     public function can($subject, $transitionName);
 
+    /**
+     * Returns transition blockers explaining why a transition cannot be made.
+     *
+     * @param object $subject        A subject
+     * @param string $transitionName A transition
+     *
+     * @return TransitionBlockerList Empty if the transition is possible
+     */
+    public function buildTransitionBlockerList($subject, string $transitionName): TransitionBlockerList;
```

Comment:
isnt this a BC break? AFAIK you are only allowed to add new interface methods in major versions

Reply:
No, because it's only in SF 4.1 which has not been released.